docs/irf/index.rst

```diff
@@ -15,10 +15,12 @@ the angular cuts.
 Energy Dispersion Matrix
 ------------------------
 
-The migration matrix, ratio of the reconstructed energy over the true energy
+The energy dispersion matrix, ratio of the reconstructed energy over the true energy
 as a function of the true energy, is computed with the events passing the
 threshold of the best cutoff plus the angular cuts.
 
+The corresponding, sparse energy migration matrix can be build from the dispersion matrix.
```

Comment:
Sparse is not part of the definion. it's more like an expected property of the migration matrix.

The real difference is in the second axis: dispersion as `(e_reco / e_true)` , migration has `e_reco`